src/modules/TalentProfileModule/Section/talentExperience.tsx

```
@@ -0,0 +1,78 @@
+import { useEffect, useState } from 'react';
```

ESLint

[error] 'useEffect' is defined but never used.

(@typescript-eslint/no-unused-vars)